pages/common/git-log.md

```diff
@@ -2,14 +2,18 @@
 
 > Show a history of commits.
 
-- Show a history of commits:
+- Show the sequence of commits leading to the current one:
```

Comment:
I am guessing you added "leading to the current one" to give a sense of the chronological ordering. I feel "leading to" is not perfectly clear on whether it starts from top or ends at the bottom. 

How about "Show sequence of commits starting from the current one" ?